NuRadioReco/utilities/geometryUtilities.py

```diff
@@ -327,7 +327,7 @@ def fresnel_factors_and_signal_zenith(detector, station, channel_id, zenith):
         t_theta = get_fresnel_t_p(zenith, n_ice, 1)
         t_phi = get_fresnel_t_s(zenith, n_ice, 1)
         logger.debug((
-            "Channel {:d}: electric field is refracted into the firn. "
+            "Channel {}: electric field is refracted into the firn. "
```

Comment:
shouldn't we start enforcing it and also convert any potential string to an integer in the reader module?

Reply:
Sure, but I'm happy for that to happen in a different PR (probably to develop), and we'll have to be careful not to break all files that have channel ids (or e.g. run numbers, etc.) in the wrong format.